### PR TITLE
Fix work modals

### PR DESCRIPTION
**Pull request by terenceingram**
The problem is that Chrome is strictly enforcing well formed HTML and Javascript standards in Bootstrap Modals.

Two main problems:

1 - The Select representative Image modal didn't have a closing </div> tag.
2 - The javascript for deleting children required scope being added to a variable.

### CODE REVIEW

**Comment by terenceingram** [comment]
@ninhnguyen @snezanam @shuangzhou @JonathanShaw Comments?

File: web/WEB-INF/views/works/show.ftl [hunk 1]
```diff
@@ -511,13 +511,13 @@
...
         <p id="delete-message"></p>
         <form role="form" id="modal-delete-form" action="works/${id}" method="POST">
-          <input type="hidden" name="_method" value="DELETE">
-          <input type="hidden" name="bibLevel" value="${bibLevel}">
-          <input type="hidden" name="hasOcr" value="${hasOcr?c}">
-          <input type="hidden" name="workId" value="${id}">
+          <input type="hidden" name="_method" value="DELETE"></input>
+          <input type="hidden" name="bibLevel" value="${bibLevel}"></input>
+          <input type="hidden" name="hasOcr" value="${hasOcr?c}"></input>
+          <input type="hidden" name="workId" value="${id}"></input>
```
Comment (JonathanShaw) [inline web/WEB-INF/views/works/show.ftl]: do these work with self closing tags? eg <input type="hidden" name="hasOcr" value="${hasOcr?c}"/> 
just a style thing though really

**Comment by JonathanShaw** [comment]
Looks good to me

File: web/WEB-INF/views/works/show.ftl (hunk 1 above)
Comment (terenceingram) [inline web/WEB-INF/views/works/show.ftl]: That's interesting. On that template we have a mix of:
* not closed <input ...> 
* closed <input .... />
* closing tag <input ...></input>

It appears that the Modal being rendered is very strict with HTML. Not sure why.

File: web/WEB-INF/views/works/show.ftl [hunk 2]
```diff
@@ -617,6 +617,7 @@
                 <div class="col-md-1">
                   <button id="search-representative-image" class="btn btn-success btn-xs navbar-btn btn-block pull-right" onclick="javascript:validateImgWork();event.preventDefault();"><i class="glyphicon glyphicon-search"></i></button> 
                 </div>
+              </div>
```
Comment (shuangzhou) [inline web/WEB-INF/views/works/show.ftl]: Thanks for finding the missing </div> tag.

**Pull request merged**